ktor-server/ktor-server-servlet/src/io/ktor/server/servlet/ServletUpgrade.kt

```diff
@@ -94,6 +94,8 @@ class ServletUpgradeHandler : HttpUpgradeHandler, CoroutineScope {
         launch(up.userContext, start = CoroutineStart.UNDISPATCHED) {
             val job = up.upgradeMessage.upgrade(inputChannel, outputChannel, up.engineContext, up.userContext)
 
+            // TODO it's parent-child?
```

Comment:
Why are you not using attachChild API here?

Reply:
After discussion with Roman I'd remove it after beta release